recipes/header/settings_menu_button/settings_menu_button.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <dt-button
+    v-if="updateAvailable"
+    importance="outlined"
+    size="xs"
+    icon-position="right"
+    class="d-h32 d-bar-pill d-bgc-blue-200 d-bgo25 h:d-bgo50 d-bc-blue-500 d-bco25 d-fc-blue-500"
+    :aria-label="ariaLabel"
+  >
+    <slot />
+    <template #icon>
+      <dt-icon
+        name="more-vertical"
+        size="400"
+        class="d-fc-blue-500"
+      />
+    </template>
+  </dt-button>
+  <dt-button
+    v-else
+    importance="clear"
+    kind="inverted"
+    class="settings-menu-button d-p0 d-bar24 d-h32 d-w24 d-theme-topbar-fc"
+    :aria-label="ariaLabel"
+  >
+    <template #icon>
+      <dt-icon
+        name="more-vertical"
+        size="400"
+      />
+    </template>
+  </dt-button>
+</template>
+
+<script>
+import DtButton from '@/components/button/button';
+import { DtIcon } from '@/components/icon';
+
+export default {
+  name: 'SettingsMenuButton',
+
+  components: {
+    DtButton,
+    DtIcon,
+  },
+
+  props: {
+    /**
+     * Determines whether the button should display the update state
+     * default is false
+     * @values true, false
+     */
+    updateAvailable: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Aria label
+     */
+    ariaLabel: {
+      type: String,
+      required: true,
+    },
+  },
+
+  emits: [
+    /**
+     * Native click event
+     *
+     * @event click
+     * @type {PointerEvent | KeyboardEvent}
+     */
+    'click',
+  ],
```

Comment:
Remove the emits here, unlike in vue 2 the emits in vue 3 means you are manually emitting a custom event via `this.$emit`. So it needs to be removed when an event is emitted natively. This sucks a bit for documentation but you can add it directly into the stories.js.